packages/devtools/src/graphQlDirectives/ceramicStringType.ts

```diff
@@ -0,0 +1,44 @@
+import { GraphQLScalarType, GraphQLString, ValueNode } from 'graphql'
+
+export class CeramicStringType extends GraphQLScalarType<string> {
```

Comment:
Do we need a custom scalar type to handle the directive please? Can't we set the directive on a `String`?